src/components/LocationsDrop/LocationsDrop.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect } from "react";
+
+import { Label, Dropdown, FormGroup } from "@trussworks/react-uswds";
+import { connect } from "react-redux";
+import {
+  setLocationSelectionState,
+  setInactiveToggle,
+} from "../../store/actions/dynamicFacilityTab";
+import { setActiveTab } from "../../store/actions/activeTab";
+const LocationDrop = ({
+  orisCode,
+  caption,
+  options,
+  selectKey,
+  showInactive = false,
+  initialSelection,
+  monitoringPlans,
+  setLocation,
+  setActiveTab,
+  tabs,
+  activeTab,
+}) => {
+  const getIndex = (val) => {
+    return options.findIndex((obj) => obj.id === val);
+  };
+
+  const [selectionState, setSelectionState] = useState(
+    tabs[activeTab[0]].location[0]
+  );
+
+  const handleChange = (val) => {
+    setSelectionState(getIndex(val.target.value));
+    setLocation([getIndex(val.target.value), val.target.value], orisCode);
+  };
+
+  const populateOptions = (optionsList) => {
+    return optionsList.map((info, index) => {
+      return (
+        <option key={info.id} value={info.id}>
+          {info[selectKey]}
+        </option>
+      );
+    });
+  };
+
+  useEffect(() => {
+  
+    if (initialSelection >= 0 && tabs[activeTab[0]].locations.length >= 1) {
+      // setSelectionState(
```

Comment:
do we plan on using this code again at some point, or can it be deleted (remember we still have access to it through versioning)